module-4/backend/week_four.md

```diff
@@ -1,19 +1,41 @@
 ## Week Four - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week.
 
-Note: When you're done, submit a PR. 
+Note: When you're done, submit a PR.
 
 1. What's your favorite project management tool?
+> From all of the ones that we have tried while at turing I'll say Pivotal tracker is the best one so far.
 2. Why do we create staging environments?
+> It's the closes thing to a production look while still in local, so it's easier to debug.
```

Comment:
it's not technically local - it's hosted on a third party service, just like production is. the goal is to mimic production as closely as possible 👍